tutorials/applications/QAA to solve a MWIS problem.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
<p>On the second expression, the index should be <code>1&lt;=k,j&lt;=N</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/821/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>